setup.py

```diff
@@ -13,7 +13,7 @@
     # "xevan_hash" package is required to sync Xuez network.
     # "groestlcoin_hash" package is required to sync Groestlcoin network.
     install_requires=['aiorpcX>=0.7.1,<0.8', 'attrs',
-                      'plyvel', 'pylru', 'aiohttp >= 2'],
+                      'plyvel', 'pylru', 'aiohttp >= 2', 'quark_hash'],
```

Comment:
Please remove this and import it in .travis.yml - you'll see several other coin-specific hash functions there.  With that this is good.